packages/web/tsconfig.json

```diff
@@ -1,20 +1,28 @@
 {
-  "references": [{ "path": "../core" }],
+  "$schema": "http://json.schemastore.org/tsconfig",
+  "references": [{ "path": "../core" }, { "path": "../react" }],
   "compilerOptions": {
+    "checkJs": true,
     "module": "node16",
-    "rootDir": "src",
-    "target": "es2021",
+    "target": "esnext",
     "sourceMap": true,
     "declaration": true,
     "declarationMap": true,
     "incremental": true,
-    "outDir": "dist/",
+    "emitDeclarationOnly": true,
+    "composite": true,
     "strict": true,
     "isolatedModules": true,
     "skipLibCheck": true,
-    "experimentalDecorators": true,
-    "useDefineForClassFields": false,
     "verbatimModuleSyntax": true,
-    "plugins": [{ "name": "ts-lit-plugin" }]
-  }
+    "jsx": "react",
+    "outDir": "dist/",
+    "rootDir": "src/",
+    "tsBuildInfoFile": ".tsbuildinfo",
+    "paths": {
+      "react": ["./node_modules/preact/compat/"],
+      "react-dom": ["./node_modules/preact/compat/"]
+    }
+  },
+  "include": ["src/*"]
```

Comment:
This is already implied by `rootDir`.

```suggestion
```

Reply:
It is not. If I remove it, I get errors like

```sh
error TS6059: File '/Users/nick/dev/motif.land/markprompt-js/packages/web/scripts/dev.js' is not under 'rootDir' '/Users/nick/dev/motif.land/markprompt-js/packages/web/src'. 'rootDir' is expected to contain all source files.
  The file is in the program because:
    Matched by default include pattern '**/*'
  File is ECMAScript module because '/Users/nick/dev/motif.land/markprompt-js/packages/web/package.json' has field "type" with value "module"
```